providers/class.two-factor-fido-u2f-admin.php

```diff
@@ -56,6 +56,18 @@ public static function enqueue_assets( $hook ) {
 		wp_enqueue_script( 'fido-u2f-admin', plugins_url( 'js/fido-u2f-admin.js', __FILE__ ), array( 'jquery', 'u2f-api' ), null, true );
 
 		$user_id = get_current_user_id();
+		if ( 'user-edit.php' == $hook ) {
+			if ( empty( $_POST['user_id'] ) ) {
+				if ( empty( $_GET['user_id'] ) ) {
+					$user_id = '';
+				} else {
+					$user_id = $_GET['user_id'];
+				}
+			} else {
+				$user_id = $_POST['user_id'];
```

Comment:
@shield-9 We can't rely on the user_id passed in the request. This could allow `contributors` to add security keys to other users.